### ISSUE TITLE
Play again addon 

### ISSUE THREAD

**Issue by tomkeuper** [tomkeuper/BedWars1058#135]
Adds a play again feature to BedWars1058, allowing players to quickly start another game without returning to the lobby

- [x] Approval from original dev
- [ ] Implement into base plugin
- [ ] Test features

https://polymart.org/resource/1946/

**Comment by tomkeuper** [comment]
We need to add credits to Cube. But implementation has been approved.

**Comment by KKiiya** [comment]
Cool, but about this, Cube said that the plugin couldnt work properly with bwproxy (idk what is it) and it doesnt work how it should

**Comment by porudev** [comment]
tom we need to address the elephant in the room

**Comment by tomkeuper** [comment]
> tom we need to address the elephant in the room

This addon is not easy to implement and will require some form of communication between a proxy server and the game server, and as that system still needs to be improved its easier said than done

**Comment by CubeCrafter72** [comment]
yeah, the bungee mode in my addon was kinda buggy and unstable because of the way bw1058 works, so i decided to remove it completely from the addon.
I hope you can implement it in a better way

**Comment by tomkeuper** [comment]
> yeah, the bungee mode in my addon was kinda buggy and unstable because of the way bw1058 works, so i decided to remove it completely from the addon. I hope you can implement it in a better way

I was planning on implementing Redis as a messaging service. Addons will be able to hook into this, would that solve most of the issues you were facing?

**Comment by CubeCrafter72** [comment]
> > yeah, the bungee mode in my addon was kinda buggy and unstable because of the way bw1058 works, so i decided to remove it completely from the addon. I hope you can implement it in a better way
> 
> I was planning on implementing Redis as a messaging service. Addons will be able to hook into this, would that solve most of the issues you were facing?

I think so, but some of the arena joining/leaving logic would have to be changed to make it work properly
